alacritty_terminal/src/term/mod.rs

```diff
@@ -706,14 +712,26 @@ impl VisualBell {
                 // Since we want the `intensity` of the VisualBell to decay over
                 // `time`, we subtract the `inverse_intensity` from 1.0.
                 1.0 - inverse_intensity
-            },
+            }
+        }
+    }
+
+    pub fn run_command(&self) {
```

Comment:
This function name does not correctly represent what is done inside the function. A generic command execution should not print `bell` errors.

I'd recommend inlining this into the `run` method.

Reply:
I personally don't mind a `Bell::run_command` function like this, though I don't care much either way since `Bell::ring` is so short.

If this were to be left as it's own function it could be hidden as an internal detail by removing the `pub` declaration.